hazelcast/src/main/java/com/hazelcast/instance/AbstractMember.java

```diff
@@ -16,15 +16,16 @@
 
 package com.hazelcast.instance;
 
-import com.hazelcast.nio.serialization.impl.BinaryInterface;
 import com.hazelcast.cluster.MemberAttributeOperationType;
 import com.hazelcast.core.Member;
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.nio.Address;
```

Comment:
This class is used as a parent class on the client side. Are we sure this isn't going to cause wire incompatibilities?